CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Minor: Added `/popout` command. Usage: `/popout [channel]`. It opens browser chat for the provided channel. Can also be used without arguments to open current channels browser chat. (#2556, #2812)
 - Minor: Improved matching of game names when using `/setgame` command (#2636)
 - Bugfix: Fixed FFZ emote links for global emotes (#2807, #2808)
+- Bugfix: Fix reconnecting when IRC write connection is lost
```

Comment:
```suggestion
- Bugfix: Fix reconnecting when IRC write connection is lost (#1831, #2356, #2850)
```
adding issue #1831, #2356 and this pr to the changelog

Reply:
also referencing #2692, even though you stated you were working on the reconnection aspect